### ISSUE TITLE
Menu items duplicate after disable() or destroy()

### ISSUE THREAD

**Issue by juxuanu** [juxuanu/cpupower-governors#4]
Coming from suspend, lock, etc. Menus duplicate. I'm trying to fix it but I am pretty new to Gnome Extensions. If someone knows how to fix it, mention it here or PR.

**Comment by Thesola10** [comment]
I tried disabling the extension and the menu entries remain, which leads me to think that the extension doesn't destroy menu elements when it should, rather than outright "duplicating them".

**Comment by juxuanu** [comment]
Yeah, it's a mess. I'll put some time in the future and implement a better version of this. My plans are to have a toggle for boost, a selector for scheduler and a battery charge threshold in the menu. No dependencies will be needed.
